data/recognizer_registry/QUT/data.yml

```diff
@@ -778,32 +778,73 @@
         tn: 0
         fn: 3616
 
-  repository: https://github.com/QutEcoacoustics/audio-analysis/blob/master/src/AnalysisConfigFiles/RecognizerConfigFiles/Truskinger.ManorinaMelanophrys.yml
+
+- title: Kookaburra Recognizer
+  description: Recognizer for the Laughing Kookaburra (Dacelo novaeguineae)
+  targets:
+    - name: Kookaburra
+      binomial: Dacelo
```

Comment:
description and bionomial name are inconsistent (is this a genus recognizer or a species recognizer?)